src/Pyz/Yves/ProductConfiguratorGatewayPage/ProductConfiguratorGatewayPageDependencyProvider.php

```diff
@@ -28,6 +34,25 @@ class ProductConfiguratorGatewayPageDependencyProvider extends SprykerProductCon
     protected function getProductConfiguratorRequestPlugins(): array
     {
         return [
+            new class implements ProductConfiguratorRequestStrategyPluginInterface {
```

Comment:
So this plugin and one below (in `getProductConfiguratorResponsePlugins`) are stubs that need to be made much more elaborate depending on the requirements. But here you can see that the access token requests are part of the Example configurator use case, all that is not required for conf product.